sale_packaging_price/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Packaging Price
+====================
+
+This module was written to extends the functionality of sale workflow
+and allow you to assign a selling price of a complete package.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* go to warehouse config and check 'Use packages: pallets, boxes, ...' and
+'Allow to define several packaging methods on products'
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to product and into procurement tab you can add new packages with a
```

Comment:
Ok.
About the previous comment(100.02) you can increment the price decimal precision.